show/main.py

```diff
@@ -62,6 +62,7 @@
 from . import system_health
 from . import warm_restart
 from . import plugins
+from . import radius
```

Comment:
Alternatively - we can import only if RADIUS is enabled (move under 'if' statement, line 213)